frontend/components/dashboard/CollateralTable.vue

```diff
@@ -11,7 +11,7 @@
             <CurrencyIcon :currency-symbol="record.symbol" />
         </div>
         <div slot="ilk" slot-scope="ilk" class="Element">
-            {{ ilk }}
+            {{ ilk }} <span v-if="isCollateralOffboarded(ilk)" class="opacity-50">&nbsp;(off-boarded)</span>
```

Comment:
Nit: we might just use a separate column for that with `off-boarded`/`active` label